canonical-websites/www.canonical.com
# Update feeds

## Done

- Fix broken feeds on about page

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8002/about](http://0.0.0.0:8002/about)
4. Make sure news and resources feeds have content and do not display feed not loaded message


## Issue / Card

Fixes #244
